js/whitebit.js

```diff
@@ -615,11 +708,439 @@ module.exports = class whitebit extends Exchange {
+        //     "timestamp": 1595792396.165973,    // current timestamp
+        //     "dealMoney": "0",                  // if order finished - amount in money currency that finished
+        //     "dealStock": "0",                  // if order finished - amount in stock currency that finished
+        //     "amount": "0.001",                 // amount
+        //     "takerFee": "0.001",               // maker fee ratio. If the number less than 0.0001 - it will be rounded to zero
+        //     "makerFee": "0.001",               // maker fee ratio. If the number less than 0.0001 - it will be rounded to zero
+        //     "left": "0.001",                   // if order not finished - rest of amount that must be finished
+        //     "dealFee": "0",                    // fee in money that you pay if order is finished
+        //     "price": "40000",                  // price
+        //     "activation_price": "40000"        // activation price -> only for stopLimit, stopMarket
+        // }
+        // For Closed Orders
+        //
+        //     {
+        //         "market": "BTC_USDT"             // artificial field
+        //         "amount": "0.0009",               // amount of trade
+        //         "price": "40000",                 // price
+        //         "type": "limit",                  // order type
+        //         "id": 4986126152,                 // order id
+        //         "clientOrderId": "customId11",    // custom order identifier; "clientOrderId": "" - if not specified.
+        //         "side": "sell",                   // order side
+        //         "ctime": 1597486960.311311,       // timestamp of order creation
+        //         "takerFee": "0.001",              // maker fee ratio. If the number less than 0.0001 - its rounded to zero
+        //         "ftime": 1597486960.311332,       // executed order timestamp
+        //         "makerFee": "0.001",              // maker fee ratio. If the number less than 0.0001 - its rounded to zero
+        //         "dealFee": "0.041258268",         // paid fee if order is finished
+        //         "dealStock": "0.0009",            // amount in stock currency that finished
+        //         "dealMoney": "41.258268"          // amount in money currency that finished
+        //     },
+        const marketId = this.safeString (order, 'market');
+        const symbol = this.safeSymbol (marketId, market, '_');
+        const side = this.safeString (order, 'side');
+        const filled = this.safeValue (order, 'dealStock');
+        const amount = this.safeValue (order, 'amount');
+        const clientOrderId = this.safeValue (order, 'clientOrderId');
+        const price = this.safeValue (order, 'price');
+        const activationPrice = this.safeValue (order, 'activation_price');
+        let orderId = this.safeValue (order, 'orderId');
+        if (orderId === undefined) {
+            orderId = this.safeValue (order, 'id');
```

Comment:
Let's always use `safeString` for `fee['cost']`, `amount`, `filled`, `price`, `stopPrice`, `orderId`.